exercises/resistor-color-duo/.meta/src/reference/java/ResistorColorDuo.java

```diff
@@ -0,0 +1,12 @@
+/**
+ * Example Class
+ */
```

Comment:
You should remove this comment also. It is clear from the folder structure that this is the reference implementation.